Gemfile.lock

```diff
@@ -646,7 +643,7 @@ GEM
     xpath (2.0.0)
       nokogiri (~> 1.3)
     zonebie (0.6.1)
-    zxcvbn-js (4.4.1)
+    zxcvbn-js (4.4.2)
```

Comment:
Since we're storing the JS file in our app, this probably means we also need to update the file in `node_modules/zxcvbn/dist/zxcvbn.js`, right?